applications/MeshMovingApplication/tests/test_linear_transform.py

```diff
@@ -0,0 +1,64 @@
+import KratosMultiphysics
+import KratosMultiphysics.MeshMovingApplication as MeshMoving
+import KratosMultiphysics.KratosUnittest as UnitTest
+
+import numpy
```

Comment:
If possible avoid numpy. From what I see here the functionalities can be taken over by the standard python math.

For now we are aiming to avoid additional dependencies and numpy as not standard for KratosMultiphysics.

Reply:
Explanation:
When packaging Kratos (e.g. for th GiD per/postprocessor) then any non-standard packages make it hard / even impossible sometimes to package it

Furthermore some systems might not have this package, then this test would fail with an `ImportError`
=> ANY non-standard module imports must be protected in the tests and the corresponding test must be skipped

[Example](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/CoSimulationApplication/tests/test_CoSimulationApplication.py#L4-L8) (I just realized it is not done perfectly there either but you get the idea :) )

Reply:
Makes sense. I usually try to avoid python math because it often has weird/unintuitive behaviour with absolute values and integer/floating point conversions, but that's obviously not relevant here.